content/projekte/_index.md

```diff
@@ -9,16 +9,88 @@ outputs:
 
 {{< headline-icon icon="icon-documents.svg" color="blue" >}}
 
-{{< headline-brackets-h1 color="red"  >}}
-Unsere Projekte
-{{< /headline-brackets-h1  >}}
+{{< headline-brackets-h1 color="red" >}}
+Projekte
+{{< /headline-brackets-h1 >}}
 
+{{< hero-block-center >}}
+[Alle Projekte](projekte/alle), die bei Code for Germany entstehen, sind Open Source. Sie lassen sich kostenlos verwenden und weiterentwicklen.
```

Comment:
Link falsch:
[Alle Projekte](https://deploy-preview-425--hungry-heisenberg-4bd67f.netlify.app/projekte/projekte/alle), die bei Code for Germany...

sollte sein `[Alle Projekte](alle)`

Komischerweise funktioniert weiter unten im Text ` (/projekte/alle)` dann bei "Du kannst dir gleich [alle Projekte](https://deploy-preview-425--hungry-heisenberg-4bd67f.netlify.app/projekte/alle) ansehen", scheint also eher ein Makro-Problem zu sein?